tests/MongoDB.Driver.Tests/Specifications/Runner/MongoClientJsonDrivenSessionsTestRunner.cs

```diff
@@ -25,7 +25,7 @@ public abstract class MongoClientJsonDrivenSessionsTestRunner : MongoClientJsonD
     {
         private const string SessionIdKeySuffix = "__ClientSessionId";
 
-        protected override string[] ExpectedTestColumns => new[] { "description", "clientOptions", "failPoint", "sessionOptions", "operations", "expectations", "outcome", "async" };
+        protected override string[] ExpectedTestColumns => new[] { "description", "clientOptions", "useMultipleMongoses", "failPoint", "sessionOptions", "operations", "expectations", "outcome", "async" };
```

Comment:
Very minor comment: if this we get a lot more columns in the future, perhaps using a HashSet may become more efficient: running some quick tests locally, the array still seems to be faster now :)